my_type=hypothesis
proj=hdfs
u_test=org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list[1]=dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 23348344997
---> v1v2 test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 21921808720
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 21554621156
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 24942581404
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
vv_mode: v1v2
result: FAIL
failureMessage: new checkpoint does not exist
stackTrace: java.lang.AssertionError: new checkpoint does not exist
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.verifyNNCheckpoint(TestRollingUpgrade.java:671)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpoint(TestRollingUpgrade.java:648)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpointWithMultipleNN(TestRollingUpgrade.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 19385229507
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 22583729689
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 23244397151
---> v1v2 test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 20536923326
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 19090992067
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 21594235711
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
vv_mode: v1v2
result: FAIL
failureMessage: new checkpoint does not exist
stackTrace: java.lang.AssertionError: new checkpoint does not exist
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.verifyNNCheckpoint(TestRollingUpgrade.java:671)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpoint(TestRollingUpgrade.java:648)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpointWithMultipleNN(TestRollingUpgrade.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 25173674973
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 21126015748
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 22648647012
---> v1v2 test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 23329809969
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 22224616320
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 25349154778
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
vv_mode: v1v2
result: FAIL
failureMessage: new checkpoint does not exist
stackTrace: java.lang.AssertionError: new checkpoint does not exist
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.verifyNNCheckpoint(TestRollingUpgrade.java:671)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpoint(TestRollingUpgrade.java:648)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpointWithMultipleNN(TestRollingUpgrade.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 21506665698
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 17237772538
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 18379422907
---> v1v2 test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 21789878288
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 17722745783
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 23881770070
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
vv_mode: v1v2
result: FAIL
failureMessage: new checkpoint does not exist
stackTrace: java.lang.AssertionError: new checkpoint does not exist
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.verifyNNCheckpoint(TestRollingUpgrade.java:671)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpoint(TestRollingUpgrade.java:648)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpointWithMultipleNN(TestRollingUpgrade.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 22402352919
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 16421631928
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 24791923767
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
vv_mode: v1v2
result: FAIL
failureMessage: new checkpoint does not exist
stackTrace: java.lang.AssertionError: new checkpoint does not exist
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.verifyNNCheckpoint(TestRollingUpgrade.java:671)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpoint(TestRollingUpgrade.java:648)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpointWithMultipleNN(TestRollingUpgrade.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 20421704412
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 22412456683
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 24720095893
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
vv_mode: v1v2
result: FAIL
failureMessage: new checkpoint does not exist
stackTrace: java.lang.AssertionError: new checkpoint does not exist
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.verifyNNCheckpoint(TestRollingUpgrade.java:671)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpoint(TestRollingUpgrade.java:648)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpointWithMultipleNN(TestRollingUpgrade.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 21995606800
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 21424521809
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 22518144847
---> v1v2 test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 21128299105
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 22039454455
---> v1v1 and v2v2 test test suceeded.
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v2 running time = 20208423411
---> v1v2 test failed.
proj: hdfs
u_test: org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
h_list: dfs.ha.tail-edits.period.backoff-max@@@hdfs:NameNode@@@-2@@@100s@@@0
vv_mode: v1v2
result: FAIL
failureMessage: new checkpoint does not exist
stackTrace: java.lang.AssertionError: new checkpoint does not exist
	at org.junit.Assert.fail(Assert.java:88)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.verifyNNCheckpoint(TestRollingUpgrade.java:671)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpoint(TestRollingUpgrade.java:648)
	at org.apache.hadoop.hdfs.TestRollingUpgrade.testCheckpointWithMultipleNN(TestRollingUpgrade.java:571)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:298)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:292)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)


updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v1v1 running time = 21291675218
updating test result for file org.apache.hadoop.hdfs.TestRollingUpgrade#testCheckpointWithMultipleNN
v2v2 running time = 21468480908
---> v1v1 and v2v2 test test suceeded.
early stop after 7 is satisfied
v1v2 failed with probability 7 out of 12
v1v1v2v2 failed with probability 0 out of 12
Total execution time in seconds : 781
0
